src/release/release.ts

```diff
@@ -680,12 +700,46 @@ export class Release extends Component {
         preBuildSteps,
         task: releaseTask,
         postBuildSteps,
+        jobDefaults: this.project.topLevelParent
+          ? // If this is a subproject, we need to set the working directory to the outdir of the subproject
+            { run: { workingDirectory: `./${this.project.relativeOutdir}` } }
+          : undefined,
         ...filteredRunsOnOptions(this.workflowRunsOn, this.workflowRunsOnGroup),
       });
+
+      workflow.addJob(BUILD_JOBID, taskjob);
+
+      return workflow;
     } else {
       return undefined;
     }
   }
+
+  private _findTargetGitHubForWorkflow(): GitHub | undefined {
```

Comment:
Trying to get away from this convention. I know it's not consistently enforced.

```suggestion
  private findTargetGitHubForWorkflow(): GitHub | undefined {
```